tktitler.py

```diff
@@ -639,7 +639,34 @@ def parse(alias, gfyear=None):
     return root, gfyear - age
 
 
-def _validate_title(title):
+def validate_title(title):
+    """
+    Givet en titel af (root, period), validerer om det er en gyldig titel. Kan raise TypeError eller ValueError.
```

Comment:
Hmm, jeg tror måske jeg synes vi burde begrænse det til ValueError (og ændre den nuværende opførsel når man stopper forkerte typer ind).

Reply:
Erstatte de to nuværende `TypeError`s med `ValueError`?

Reply:
Ping @Mortal

Reply:
Ja, erstatte de to nuværende TypeErrors med ValueError.